src/tests/end-to-end/common/element-identifiers/popup-page-element-identifiers.ts

```diff
@@ -10,4 +10,5 @@ export const popupPageElementIdentifiers = {
     backToLaunchPadLink: '#back-to-launchpad-link',
     hamburgerMenuButton: '#feedback-collapse-menu-button',
     hamburgerMenu: '.popup-menu',
+    fastPass: '#fast-pass',
```

Comment:
We aren't using any of the fastpass selectors anymore (here or in `fastpass-selectors.ts`), so I'd be inclined to remove them from the PR; every selector we're encoding in the E2E tests is an opportunity to become out of date with the main body of code, especially if there isn't an active test using them.

Reply:
removed.